src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/idam/cache/IdamCachedClient.java

```diff
@@ -39,8 +39,8 @@ public IdamCachedClient(
     }
 
     public CachedIdamCredential getIdamCredentials(String jurisdiction) {
-        log.info("Getting idam credential for jurisdiction: {} ", jurisdiction);
-        return this.idamCache.get(jurisdiction, this::retrieveIdamInfo);
+        log.info("Getting idam credential for jurisdiction: {} ", jurisdiction.toLowerCase());
```

Comment:
In log entry we can probably leave the original jurisdiction. Just a suggestion.